### PR TITLE
JBPM-4917: Error when deploying Workbench to WebSphere8.5

### PR DESCRIPTION
**Pull request by dgutierr**
Fix + test

### CODE REVIEW

File: uberfire-security/uberfire-security-management/uberfire-security-management-backend/src/main/java/org/uberfire/ext/security/management/BackendUserSystemManager.java [hunk 1]
```diff
@@ -92,6 +91,8 @@ public void initialize() throws Exception {
                     isUserManagerActive = true;
                 } catch (ClassCastException e) {
                     // Manager is not contextual.
```
Comment (ederign) [inline uberfire-security/uberfire-security-management/uberfire-security-management-backend/src/main/java/org/uberfire/ext/security/management/BackendUserSystemManager.java]: It's not related to the PR, but swallow this exception make me nervous... wdyt to add a log here?

File: uberfire-security/uberfire-security-management/uberfire-security-management-backend/src/main/java/org/uberfire/ext/security/management/BackendUserSystemManager.java [hunk 2]
```diff
@@ -106,6 +107,8 @@ public void initialize() throws Exception {
                     isGroupManagerActive = true;
                 } catch (ClassCastException e) {
                     // Manager is not contextual.
```
Comment (ederign) [inline uberfire-security/uberfire-security-management/uberfire-security-management-backend/src/main/java/org/uberfire/ext/security/management/BackendUserSystemManager.java]: here

File: uberfire-security/uberfire-security-management/uberfire-security-management-backend/src/main/java/org/uberfire/ext/security/management/BackendUserSystemManager.java [hunk 3]
```diff
@@ -121,6 +124,8 @@ public void initialize() throws Exception {
                     isRoleManagerActive = true;
                 } catch (ClassCastException e) {
                     // Manager is not contextual.
```
Comment (ederign) [inline uberfire-security/uberfire-security-management/uberfire-security-management-backend/src/main/java/org/uberfire/ext/security/management/BackendUserSystemManager.java]: same here

File: uberfire-security/uberfire-security-management/uberfire-security-management-backend/src/main/java/org/uberfire/ext/security/management/BackendUserSystemManager.java [hunk 4]
```diff
@@ -165,15 +170,17 @@ public boolean isActive() {
...
-        
+    public void onDestroy() {
+
         if (usersManagementService != null) {
             try {
                 ContextualManager m = (ContextualManager) usersManagementService;
                 m.destroy();
             } catch (ClassCastException e) {
                 // Manager is not contextual.
-            }
```
Comment (ederign) [inline uberfire-security/uberfire-security-management/uberfire-security-management-backend/src/main/java/org/uberfire/ext/security/management/BackendUserSystemManager.java]: sa,e jere

File: uberfire-security/uberfire-security-management/uberfire-security-management-backend/src/main/java/org/uberfire/ext/security/management/BackendUserSystemManager.java [hunk 5]
```diff
@@ -182,6 +189,8 @@ public void onDestroy() throws Exception {
                 m.destroy();
             } catch (ClassCastException e) {
                 // Manager is not contextual.
```
Comment (ederign) [inline uberfire-security/uberfire-security-management/uberfire-security-management-backend/src/main/java/org/uberfire/ext/security/management/BackendUserSystemManager.java]: same here

File: uberfire-security/uberfire-security-management/uberfire-security-management-backend/src/main/java/org/uberfire/ext/security/management/BackendUserSystemManager.java [hunk 6]
```diff
@@ -191,9 +200,10 @@ public void onDestroy() throws Exception {
...
```
Comment (ederign) [inline uberfire-security/uberfire-security-management/uberfire-security-management-backend/src/main/java/org/uberfire/ext/security/management/BackendUserSystemManager.java]: and here

**Comment by ederign** [comment]
+1. @dgutierr what do you think about adding that logs?

**Comment by csadilek** [comment]
master: 2de425bb3922fd7f23ff51b70a54d482c68250ec

@romartin if you get a chance (i.e. with your next changeset), can you add some logging for the ignored exceptions? thanks.

**Comment by romartin** [comment]
@ederign @csadilek @dgutierr sure! added in my todo list, will add soon when commiting new PRs for it. Thx for the advice!

**Pull request closed without merging**